ops/benton-parity/tests/requirements.test.sh

```
@@ -0,0 +1,476 @@
+  fi
+}
+
+run_test() {
+  local name="$1"
+  shift
+  TESTS_RUN=$((TESTS_RUN + 1))
+  echo ""
+  echo "── TEST: $name ──"
+
+  # Reset evidence dir for each test
+  rm -rf "$TEST_TMPDIR"/*
```

Shellcheck (0.11.0)

[warning] Use "${var:?}" to ensure this never expands to /* .

(SC2115)